bin/main.rb

```diff
@@ -1 +1,82 @@
-puts "Hello World!"
\ No newline at end of file
+def welcome_message
+  puts 'Welcome to Tic Tac Toe Game'
+  end
+welcome_message
+
+def user_setup
+  print 'Please Enter Player One name '
+  player1 = gets.chomp
+  print 'Please Enter Player Second name '
+  player2 = gets.chomp
+
+  print "Choose #{player1} X or O: "
+  mark = gets.chomp
+  mark.upcase!
+  if mark == 'X'
+    puts "X is assigned to #{player1} and O is assigned to #{player2}"
+  else
+    puts "O is assigned to #{player1} and X is assigned to #{player2}"
+  end
+end
+
+user_setup
+
+puts 'Game Board'
+board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
```

Comment:
- [OPTIONAL] You can create an array of empty strings like this too:
`Array.new(9, '')`